### PR TITLE
don't disable building of third and fourth derivates in libxc 6.1.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

see also https://github.com/pyscf/pyscf/issues/1103

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

File: easybuild/easyconfigs/l/libxc/libxc-6.1.0-GCC-12.2.0.eb [hunk 1]
```diff
@@ -18,7 +18,12 @@ builddependencies = [
     ('Perl', '5.36.0'),
 ]
 
-local_common_configopts = "-DENABLE_FORTRAN=ON -DENABLE_FORTRAN03=ON -DENABLE_XHOST=OFF"
+local_common_configopts = "-DENABLE_FORTRAN=ON -DENABLE_FORTRAN03=ON -DENABLE_XHOST=OFF "
```
Comment (branfosj) [inline easybuild/easyconfigs/l/libxc/libxc-6.1.0-GCC-12.2.0.eb:21]:
```suggestion
local_common_configopts = "-DENABLE_FORTRAN=ON -DENABLE_XHOST=OFF "
```

`ENABLE_FORTRAN03` is no longer used:

```
CMake Warning:
  Manually-specified variables were not used by the project:

    ENABLE_FORTRAN03
```

See https://gitlab.com/libxc/libxc/-/merge_requests/258/diffs

**Review by branfosj — DISMISSED** [review]
We should cleanup the unused option while we are making a change.

**Comment by SebastianAchilles** [comment]
Since building the third and fourth derivatives takes some time: Is `parallel = 1` still needed?

After removing `parallel = 1` from the easyconfig I got these build times for `libxc-6.1.0-GCC-12.2.0.eb`:

| parallel = 1 | parallel = 64 |
| ---------------- | ------------------ |
| 4h15m22    | 25m33           |

**Comment by branfosj** [comment]
> Since building the third and fourth derivatives takes some time: Is `parallel = 1` still needed?

That has been there since it was a `ConfigureMake` in version `2.0.1` and we are now `CMakeMake` and version `6.1.0`.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/d763de6b9c11fc2db54acf6674d60f63 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
zen2-ubuntu-2304 - Linux Ubuntu 23.04 (Lunar Lobster), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.2
See https://gist.github.com/SebastianAchilles/0f9040bec8143277058c923092bf2dad for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node4111.gallade.os - Linux RHEL 8.6, x86_64, AMD EPYC 7773X 64-Core Processor (zen3), Python 3.6.8
See https://gist.github.com/boegel/11661d9bd403ce7c19f4d8ae8e3c0b65 for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/7c5cb8f0721b8456da487ff709d07028 for a full test report.

**Comment by casparvl** [comment]
Strange... it's complaining about not being able to open input files, while the build with intel-compilers completes fine. That seems impossible to me. I'll rerun, just to see if it was somehow a hickup.

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/e2c88742dd87756bd2df8d998b02f93f for a full test report.

**Comment by casparvl** [comment]
Retrying with the `buildpath` and `tmpdir` on a different fs, let's see if that helps somehow. The number of timeouts seem to be different between my two runs. Maybe our FS is just not responsive, and that's what's causing the timeouts.

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/d64fb813140c1b2f448295b1260c5599 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 525.125.06, Python 3.10.6
See https://gist.github.com/akesandgren/5d206a8c59cc138546fdfe8c45c12b7a for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 525.125.06, Python 3.10.6
See https://gist.github.com/akesandgren/210ceb602f20884089941cd64e45cbd1 for a full test report.

**Comment by boegel** [comment]
@casparvl Can you check on those failing test reports again?

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/67bf3a0f6595ad53bac72e420a0cf520 for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/724bdea50ef7f6bff26c5df5c48667bf for a full test report.

**Comment by boegel** [comment]
@casparvl ready to merge?

**Review by casparvl — APPROVED** [review]
Lgtm!

**Pull request merged**